app/download/views.py

```diff
@@ -93,3 +97,40 @@
     response.headers["X-Robots-Tag"] = "noindex, nofollow"
 
     return response
+
+
+@download_blueprint.route(
+    "/services/<uuid:service_id>/documents/scan-verdict/<uuid:document_id>", methods=["GET"]
+)
+def check_scan_verdict(service_id, document_id):
+    sending_method = request.args.get("sending_method", "link")
+    try:
+        av_status = scan_files_document_store.check_scan_verdict(service_id, document_id, sending_method)
+    except (MaliciousContentError, SuspiciousContentError) as e:
+        current_app.logger.info(
+            "Malicious content detected, refused to download document: {}".format(e),
+            extra={
+                "service_id": service_id,
+                "document_id": document_id,
+            },
+        )
+        return jsonify(error=str(e)), 403
```

Comment:
As a 403 is quite standard for unauthorized access, it might be a bit misleading. The API has access but we refuse to send it back. The `423 - locked` status could be more aligned, or we can use a number that isn't in use with our our definition.